docs/distributed/README.md

```diff
@@ -43,25 +43,25 @@ To start a distributed MinIO instance, you just need to pass drive locations as
 *Note*
 
 - All the nodes running distributed MinIO need to have same access key and secret key for the nodes to connect. To achieve this, it is **mandatory** to export access key and secret key as environment variables, `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`, on all the nodes before executing MinIO server command.
-- All the nodes running distributed MinIO need to be in a homogeneous environment, i.e. same operating system, same number of disks and same interconnects.
-- `MINIO_DOMAIN` environment variable should be defined and exported if domain is needed to be set.
-- MinIO distributed mode requires fresh directories. If required, the drives can be shared with other applications. You can do this by using a sub-directory exclusive to minio. For example, if you have mounted your volume under `/export`, pass `/export/data` as arguments to MinIO server.
+- All the nodes running distributed MinIO setup are recommended to be in homogeneous environment, i.e. same operating system, same number of disks and same network interconnects.
```

Comment:
Should we also say that they should be same type of device like `all SSDs, HDDs or NVMes`

Reply:
Suggest a change, I can accept it here in the PR.